webhooks/virtualmachine/validation/virtualmachine_validator.go

```diff
@@ -153,6 +153,35 @@ func (v validator) ValidateDelete(*pkgctx.WebhookRequestContext) admission.Respo
 	return admission.Allowed("")
 }
 
+// Updates to VM's image are only allowed if it is a Registered VM (used for failover
+// in disaster recovery).
+func (v validator) validateImageOnUpdate(ctx *pkgctx.WebhookRequestContext, vm, oldVM *vmopv1.VirtualMachine) field.ErrorList {
+	var allErrs field.ErrorList
+
+	if f := pkgcfg.FromContext(ctx).Features; f.VMIncrementalRestore && vmopv1util.IsRegisteredVM(*vm) {
```

Comment:
If we wanna just allow unset of image/imageName. can we just check for IsImagelessVM(*vm) along with the IsRegisteredVM() check up top instead of the deepEqual() inside?. Because the deepEqual will let unset and reset now.

Reply:
I was going with the idea that if it is a failover, or import, we don't want to do _any_ validations (not necessarily check for unset/reset).

Reply:
Something to consider here. This will open up resetting image fields to new values when any privileged account modifies our VM that was imported/registered before (not just failover, right?). 

Might be safer just to allow unset (value -> "" , "" -> "") like we discussed?.